api.bs

```diff
@@ -601,12 +608,24 @@ The arguments to <a method for=PrivateAttribution>measureConversion()</a> are as
     restrictions are satisfied, this value will be encoded into the [=conversion
     report=].
   </dd>
+  <dt><dfn>maxValue</dfn></dt>
+  <dd>
+    The maximum [=conversion value=] across all contributions included in the aggregation.
+    Together with epsilon, this is used to calibrate the distribution of random noise that
+    will be added to the outcome. It is also used to determine the amount of [=privacy budget=] 
+    to expend on this [=conversion report=].
+  </dd>
   <dt><dfn>lookbackDays</dfn></dt>
   <dd>An integer number of days. Only impressions occurring within the past `lookbackDays` may match this [=conversion=].</dd>
-  <dt><dfn>ads</dfn></dt>
-  <dd>A list of [=ad identifiers=]. Only [=impressions=] having one of the listed identifiers may match this [=conversion=].</dd>
+  <dt><dfn>filterData</dfn></dt>
+  <dd>Only [=impressions=] having a [=filterData=] matching this value will be eligible to match this [=conversion=].</dd>
   <dt><dfn>impressionSites</dfn></dt>
-  <dd>A list of impression sites. Only [=impressions=] recorded by one of the impression sites are eligible to match this [=conversion=].</dd>
+  <dd>A list of impression sites. Only [=impressions=] recorded where the top-level site is on this list are eligible to match this [=conversion=].</dd>
+  <dt><dfn>intermediarySites</dfn></dt>
+  <dd>
+    A list of sites which called the [=saveImpression=] API. Only [=impressions=] recorded by scripts originating from one of the 
+    intermediary sites are eligible to match this [=conversion=].
```

Comment:
We need to connect this to the "iframe site domain" piece better than this does presently.  Issue?

Reply:
#26